src/components/users/ImportFromMtnProj.tsx

```diff
@@ -71,6 +72,9 @@ export function ImportFromMtnProj ({ isButton }: Props): JSX.Element {
             input: ticks
           }
         })
+
+        const ticksCount = ticks?.length ?? 0;
```

Comment:
I've added this kind of sanitization just in case since I don't actually know what the API returns if an abnormal situation arises.  
If it's not needed let me know and I'll just use `ticks.length`